dts/riscv/ite/it82xx2.dtsi

```diff
@@ -274,23 +274,23 @@
 			       0x00f01621 1   /* GPDMR (get) */
 			       0x00f01639 1   /* GPOTR */
 			       0x00f01651 1   /* P18SCR */
-			       0x00f016a8 8>; /* GPCR */
-			ngpios = <6>;
+			       0x00f016af 8>; /* GPCR */
```

Comment:
The register base address should be 0x00f016a8.

Reply:
Done